### PR TITLE
Updated Blaze version 3.5 and added access to master branch

### PR DESCRIPTION
**Pull request by rhewett**

### CODE REVIEW

File: var/spack/repos/builtin/packages/blaze/package.py [hunk 1]
```diff
@@ -16,8 +16,11 @@ class Blaze(Package):
     """
 
     homepage = "https://bitbucket.org/blaze-lib/blaze/overview"
-    url      = "https://bitbucket.org/blaze-lib/blaze/downloads/blaze-3.4.tar.gz"
+    url      = "https://bitbucket.org/blaze-lib/blaze/downloads/blaze-3.5.tar.gz"
+    git      = "https://bitbucket.org/blaze-lib/blaze.git"
 
+    version('master', branch='master', submodules=True, preferred=False)
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/blaze/package.py]: `preferred=False` is False by default. You don't need to add this, Spack will choose the latest numeric release by default.

Comment (rhewett) [reply var/spack/repos/builtin/packages/blaze/package.py]: I actually went to look that up and got distracted by other parts of the documentation.  Thanks for the reminder :-D.

I'll update the PR tomorrow.

Comment (rhewett) [reply var/spack/repos/builtin/packages/blaze/package.py]: Also there are no submodules so I will turn that off, too.

**Comment by rhewett** [comment]
@adamjstewart when travis is done this should be good to go now.

**Pull request merged**